### PR TITLE
Added side navbar and viewport fix

### PR DESCRIPTION
**Pull request by adesca**

### CODE REVIEW

File: competitions/index.html [hunk 1]
```diff
@@ -61,7 +63,7 @@ <h3>Jedi Swing Ep. IV: A New Hop</h3>
             </div>
         </section>
         <footer>
-            <h3>Copyright � 2017 TXISB</h3>
+            <h3>Copyright � 2017 TXISB</h3>
```
Comment (tomjw64) [inline competitions/index.html]: Seems the copyright Unicode char was replaced. Was that intended?

File: event/index.html [hunk 2]
```diff
@@ -34,6 +36,30 @@ <h2>Texas Intercollegiate Swing Battle</h2>
...
+                        <li><a href="./competitions">Competitions</a></li>
+                        <li><a href="./register">Register</a></li>
+                    </ul>
+                </nav>
+                <div class="site-logo">
+                    <img src="img/SwingBattleLogo_White.png" alt="TXISB Title"/>
+                </div>
+            </div>
+            <i class="collapsed">&#9658;</i>
+        </aside>
```
Comment (tomjw64) [inline event/index.html]: Do we need to add an extra element to do this? Can we just use JS to change how the nav bar behaves rather than adding a new aside element to do this?

File: index.html [hunk 3]
```diff
@@ -6,23 +6,42 @@
...
-                </div>
-                <div class="site-title">
-                    <img src="img/SwingBattleLogo_White.png" alt="TXISB Title" />
-                    <h1>TXISB</h1>
-                    <h2>Texas Intercollegiate Swing Battle</h2>
-                </div>
-                <div class="site-logo">
-                    <img src="img/logotransp.png" alt="TXISB Logo" />
-                </div>
+<div class="page-container">
```
Comment (tomjw64) [inline index.html]: I suppose GitHub sees this as a change because of indentatio. Since this is not vital to the feature, can we revert this change?

File: script/navbar.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            }
+        }
+    }
+});
+
+// function open() {
+//     console.log('navbar ');
+//     // state.navbarOpen = !state.navbarOpen;
+//     // var aside = document.getElementsByTagName('aside');
+// }
```
Comment (tomjw64) [inline script/navbar.js]: Is this commented code still needed? Additionally, it looks like some console debugging is still in there.

File: competitions/index.html (hunk 1 above)
Comment (adesca) [reply competitions/index.html]: That was probably windows. I assumed my computer just couldn't show it, I didn't think it'd change it.

File: event/index.html (hunk 2 above)
Comment (adesca) [reply event/index.html]: You'd have to use javascript to know the screen size, which would be kind of weird.
